spec/presenters/hyrax/admin/user_activity_presenter_spec.rb

```diff
@@ -1,21 +1,23 @@
 # frozen_string_literal: true
 RSpec.describe Hyrax::Admin::UserActivityPresenter do
-  let(:instance) { described_class.new }
+  start_date = Time.zone.today - 6.days
```

Comment:
these need to be in lets or they will break in _very_ subtle ways. For example here - start_date is set when the rspec command is first run whether or not it ever runs this spec file.